src/structures/BaseGuildTextChannel.js

```diff
@@ -30,7 +30,7 @@ class BaseGuildTextChannel extends GuildChannel {
      * If the guild considers this channel NSFW
      * @type {boolean}
      */
-    this.nsfw = Boolean(data.nsfw);
```

Comment:
I wouldn't remove the Boolean call since ur using it in VoiceChannel

Reply:
I've resolved that.